### ISSUE TITLE
Mock context separately from session/info

### ISSUE THREAD

**Issue by nshaikhinurov** [Urigo/graphql-modules#536]
Resolvers take 4 arguments: `rootValue`, `args`, `context`, `info`. I want to mock a network session which is normally in `info.session` but does not exist in Jest env. My resolver logs out a user, thus manipulates the session. The authenticated user object itself is retrieved from `context` third parameter. The problem is that I can't mock context and session separately from each other somehow like below and get user in `context.user` (3rd parameter) and session in `info.session` (4th parameter)

```javascript
// used as contextValue in execute fn
contextValue = {
    user: { id: 42, verificationStatus: 'verified' },
    session: { req: { session: {} } },
}
```

Any ideas on using new `Module.mock` API maybe?

**Comment by ardatan** [comment]
You can mock context like below;
```ts
MyModule.mock({
  contextBuilder: () => ({
     user: { id: 42, verificationStatus: 'verified' },
  })
})
```
```

**Comment by nshaikhinurov** [comment]
> You can mock context like below;
> 
> ```ts
> MyModule.mock({
>   contextBuilder: () => ({
>      user: { id: 42, verificationStatus: 'verified' },
>   })
> })
> ```

Already tried exactly this, that worked, but now i have `[[GraphQLError: Cannot read property 'callHookWithArgsAsync' of undefined]]` for uknown reason.

**Comment by nshaikhinurov** [comment]
Seems like the injector is dropped from context this way and that causes an error

**Comment by nshaikhinurov** [comment]
As a workaroud I placed my session in a `context` too, together with a user object on an authentication phase and now i do not need the `info` argument at all. Anyway thank you @ardatan for helping 👍

**Comment by ardatan** [comment]
I will work on fixing it :)
Maybe you can try to pass app injector like below until we fix it;
```ts
MyModule.mock({
  contextBuilder: () => ({
     user: { id: 42, verificationStatus: 'verified' },
     injector: MyModule.injector,
  })
})
```

**Comment by ardatan** [comment]
Could you try with the following version?
```
0.7.8-alpha-bd0befa.27+bd0befa  
```

**Comment by ardatan** [comment]
Fixed in 0.7.8!
